spec/sidekiq_jobs_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "sidekiq_jobs"
+require "sidekiq/testing"
+
+RSpec.describe "Jobs" do
```

Comment:
In general I tried to do the minimum here to ensure that the job was actually running and doing what was expected without being dependent on a lot of data. I'm not sure how much integration testing to do for the reporting here, since testing anything for real depends on enough data being loaded.